cli/network.go

```diff
@@ -0,0 +1,162 @@
+type NetworkCommand struct {
+	baseCommand
+}
+
+// Init initializes NetworkCommand command.
+func (n *NetworkCommand) Init(c *Cli) {
+	n.cli = c
+
+	n.cmd = &cobra.Command{
+		Use:   "network [command]",
+		Short: "Manage pouch networks",
+		Long:  networkDescription,
+		Args:  cobra.MinimumNArgs(1),
+	}
+
+	c.AddCommand(n, &NetworkCreateCommand{})
+}
+
+// networkCreateDescription defines the comand network create description.
+var networkCreateDescription = "Create a network in pouchd. " +
+	"It must specify network's name and driver. You can use 'network driver' to get drivers that pouch support."
+
+// NetworkCreateCommand is used to implement 'network create' command.
+type NetworkCreateCommand struct {
+	baseCommand
+
+	name       string
+	driver     string
+	gateway    string
+	ipRange    string
+	ipamDriver string
+	ipamOpts   []string
+	subnet     []string
+	options    []string
+	labels     []string
+}
+
+// Init initializes NetworkCreateCommand command.
+func (n *NetworkCreateCommand) Init(c *Cli) {
+	n.cli = c
+
+	n.cmd = &cobra.Command{
+		Use:   "create <NAME>",
+		Short: "Create a pouch network",
+		Long:  networkCreateDescription,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return n.runNetworkCreate(args)
+		},
+		Example: networkCreateExample(),
+	}
+
+	n.addFlags()
+}
+
+// addFlags adds flags for specific command.
+func (n *NetworkCreateCommand) addFlags() {
+	flagSet := n.cmd.Flags()
+
+	flagSet.StringVarP(
+		&n.name,
+		"name",
+		"n",
+		"",
+		"the name of network")
+	flagSet.StringVarP(
+		&n.driver,
+		"driver",
+		"d",
+		"bridge",
+		"the driver of network")
+	flagSet.StringVar(
+		&n.gateway,
+		"gateway",
+		"",
+		"the gateway of network")
+	flagSet.StringVar(
+		&n.ipRange,
+		"ip-range",
+		"",
+		"the range of network's ip")
+	flagSet.StringSliceVar(
+		&n.subnet,
+		"subnet",
+		nil,
+		"the subnet of network")
+	flagSet.StringVar(
+		&n.ipamDriver,
+		"ipam-driver",
+		"",
+		"the ipam driver of network")
+	flagSet.StringSliceVarP(
+		&n.options,
+		"option",
+		"o",
+		nil,
+		"create network with options")
+	flagSet.StringSliceVarP(
+		&n.labels,
+		"label",
+		"l",
+		nil,
+		"create network with labels")
+}
+
+// runNetworkCreate is the entry of NetworkCreateCommand command.
+func (n *NetworkCreateCommand) runNetworkCreate(args []string) error {
+	name := n.name
+	if len(args) != 0 {
+		name = args[0]
+	}
+	if name == "" {
+		return fmt.Errorf("need to specified network's name")
+	}
+
+	networkRequest := &types.NetworkCreateRequest{
+		Name: name,
+	}
+
+	if n.driver == "" {
+		return fmt.Errorf("need to specified network's driver")
+	}
+	networkCreate := &types.NetworkCreate{
+		Driver: n.driver,
+	}
+
+	networkRequest.NetworkCreate = networkCreate
+
+	apiClient := n.cli.Client()
+	resp, err := apiClient.NetworkCreate(networkRequest)
+	if err != nil {
+		return err
+	}
+
+	if resp.Warning != "" {
+		fmt.Printf("WARNING: %s \n", resp.Warning)
+	}
+	fmt.Printf("%s: %s\n", n.name, resp.ID)
```

Comment:
n.name should be name here, otherwise there would be the following problem:
```
[root@pouch-test monster]# pouch network create abc
: cb9eb55586fa52310ea8ab0b07a8abae47b57fd2956567a0769b92c0a8f587a2
```

Reply:
yeah, you are right.